WEB-INF/classes/gov/noaa/pfel/erddap/jte/TableOptions.java

```diff
@@ -0,0 +1,178 @@
+package gov.noaa.pfel.erddap.jte;
+
+import com.cohort.array.Attributes;
+import com.cohort.util.Calendar2;
+import com.cohort.util.MustBe;
+import com.cohort.util.String2;
+import com.cohort.util.XML;
+import gov.noaa.pfel.coastwatch.pointdata.Table;
+import java.util.List;
+
+public class TableOptions {
+  private final String otherClasses;
+  private final String bgColor;
+  private final int border;
+  private final boolean writeUnits;
+  private final int timeColumn;
+  private final boolean needEncodingAsHtml;
+  private final boolean allowWrap;
+  private final Table table;
+  private boolean somethingWritten = false;
+  private final String[] fileAccessBaseUrl;
+  private final String[] fileAccessSuffix;
+
+  public TableOptions(
```

Comment:
This is optional, but it would be nice to have a `TableOptions.Builder` to make the creation of a new `TableOptions` clearer.

https://www.digitalocean.com/community/tutorials/builder-design-pattern-in-java